### PR TITLE
OCPBUGS-23077: [release-4.12] virtual: get real PCI address for each device found

### PR DESCRIPTION
**Pull request by EmilienM**
- Update utils and create interface for better unit tests implementation
- virtual: get real PCI address for each device found
- utils_virtual: unit tests
- Add Virtio 1.0 network device as supported
- [downstream-only] Support RH Virtio 1.0 network device
- Restore `utils.LoadKernelModule(...)`
- Add gomock binary download and makefile entry to generate mocks
- Adjust `utils.GetOpenstackData(true)`

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -133,9 +136,13 @@ ENVTEST = $(shell pwd)/bin/setup-envtest
...
 
-# go-get-tool will 'go install' any package $2 and install it to $1.
 PROJECT_DIR := $(shell dirname $(abspath $(lastword $(MAKEFILE_LIST))))
-define go-get-tool
+GOMOCK = $(shell pwd)/bin/mockgen
+gomock:
+	$(call go-get-tool,$(GOMOCK),github.com/golang/mock/mockgen@v1.6.0)
+
+# go-install-tool will 'go install' any package $2 and install it to $1.
+define go-install-tool
```
Comment (zeeke) [inline Makefile:145]: Please, revert this to `go-get-tool`. There is no reference to `go-install-tool` in this file, for release-4.12

Comment (EmilienM) [reply Makefile]: ok I've removed the commit and I ran `go mod tidy && go mod vendor`. I hope this is fine.

**Comment by EmilienM** [comment]
ping @zhaozhanqi @zeeke @bn222 @wizhaoredhat please take a look :)

**Comment by EmilienM** [comment]
@bn222 for review, when time permits. Thanks

**Pull request merged**